Applications/Spire/Source/KeyBindings/TaskKeysPage.cpp

```diff
@@ -73,18 +87,155 @@ TaskKeysPage::TaskKeysPage(std::shared_ptr<KeyBindingsModel> key_bindings,
   auto layout = make_vbox_layout(body);
   layout->addWidget(make_help_text_box());
   layout->addWidget(toolbar);
-  layout->addWidget(make_task_keys_table_view(
+  m_table_view = make_task_keys_table_view(
     m_key_bindings->get_order_task_arguments(),
     std::make_shared<LocalComboBoxQueryModel>(), std::move(destinations),
-    std::move(markets)));
+    std::move(markets));
+  m_table_view->installEventFilter(this);
+  layout->addWidget(m_table_view);
   auto box = new Box(body);
   update_style(*box, [] (auto& style) {
     style.get(Any()).set(BackgroundColor(QColor(0xFFFFFF)));
   });
   enclose(*this, *box);
+  m_new_task_form = new LineInputForm(tr("New Task"), *this);
```

Comment:
Can you only create this form when it's needed and used?

Reply:
Sure.